dockstore-webservice/src/main/java/io/dockstore/webservice/core/LambdaEvent.java

```diff
@@ -36,6 +36,7 @@
     @NamedQuery(name = "io.dockstore.webservice.core.LambdaEvent.findByOrganization", query = "SELECT lambdaEvent FROM LambdaEvent lambdaEvent WHERE lambdaEvent.repository like :organization"),
     @NamedQuery(name = "io.dockstore.webservice.core.LambdaEvent.findByUsername", query = "SELECT lambdaEvent FROM LambdaEvent lambdaEvent WHERE lambdaEvent.githubUsername = :username"),
     @NamedQuery(name = "io.dockstore.webservice.core.LambdaEvent.findByUser", query = "SELECT lambdaEvent FROM LambdaEvent lambdaEvent WHERE lambdaEvent.user = :user"),
+    @NamedQuery(name = "io.dockstore.webservice.core.LambdaEvent.findOrganizationsWithEvents", query = "select distinct(lambdaEvent.organization) FROM LambdaEvent lambdaEvent WHERE lambdaEvent.organization in :organizations")
```

Comment:
devil's advocate: related to top-level comment, maybe we should just get all orgs regardless of lambda events because that seems like the next logical question (what orgs does dockstore think I have access to?)